src/common/transformations/src/transformations/common_optimizations/remove_useless_convert_like.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/remove_useless_convert_like.hpp"
+
+#include <memory>
+#include <vector>
+
+#include "itt.hpp"
+#include "openvino/core/rt_info.hpp"
+//#include <ngraph/pattern/op/wrap_type.hpp>
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "transformations/utils/utils.hpp"
+
+using namespace ov::op;
+
+ov::pass::RemoveUselessConvertLike::RemoveUselessConvertLike() {
+    MATCHER_SCOPE(RemoveUselessConvertLike);
+
+    const auto const_pattern = pattern::any_input();
```

Comment:
Not sure what is the original intention, but the word "const" here looks confusing as I'd expect Constant op in the pattern, but it doesn't exits.

Reply:
agree, old name was confusing. in the current soslution i removed any c++ changes